### PR TITLE
update docs to include manual deployments

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn @bruskiza review please
This is what I have so far - probably needs a _few more words_, but so far it covers all that's needed to run locally and deploy to production.

File: README.rst [hunk 1]
```diff
@@ -26,14 +31,35 @@ Initialize the database using::
...
 
-Then run the server usinng::
+Then run the server using (runs on http://localhost:8000)::
 
     $ ./manage.py runserver
 
+In order to access the admin interface, you'll need to create a super user::
+
+    $ ./manage.py createsuperuser
+    Then navigate to http://localhost:8000/admin/ once the server is up and running
```
Comment (smn) [inline README.rst]: You probably want to remove this indent, otherwise it'll get the same `code` formatting.

File: README.rst [hunk 2]
```diff
@@ -26,14 +31,35 @@ Initialize the database using::
...
+
+
+Deploying to Production
+=======================
+
+We use fabric to manage our deployments. Make sure your ssh keys are added to the server and use the following fabric command `fab <env>:<username> <command>` e.g::
+
+    fab production:miltontony deploy
```
Comment (smn) [inline README.rst]: Looking at the fabfile this should probably be `prod`.

File: fabfile.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        print 'format: fab <env>:<username> <command>'
+        print ' e.g fab prod:ubuntu push'
+        raise RuntimeError('Username required')
+
+    env.hosts = ['%s@a4w.ca' % ssh_user]
+
+
+def push():
+    with cd(env.path):
+        sudo('git pull', user=env.sudo_user)
```
Comment (smn) [inline fabfile.py:31]: Should this method perhaps be called `pull`?

File: fabfile.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        sudo(
+            '%(ve)s/bin/python manage.py collectstatic --noinput' % env,
+            user=env.sudo_user)
+
+
+def migrate(app=''):
+    with cd(env.path):
+        env.app = app
+        sudo(
+            '%(ve)s/bin/python manage.py migrate %(app)s' % env,
```
Comment (smn) [inline fabfile.py]: You probably want a `--noinput` here too.

**Comment by smn** [comment]
Minor comments, :+1: in general.

**Pull request merged**